docs/guide.md

```diff
@@ -0,0 +1,103 @@
+# User Guide
+
+## Session
+
+Communication with the Planet services is provided with the `Session` class.
+The recommended way to use a `Session` is as a context manager. This will
+provide for automatic clean up of connections when the context is left.
+
+```python
+>>> import asyncio
+>>> import os
+>>> from planet import Session
+>>>
+>>> AUTH = (os.getenv('PL_API_KEY'), '')
+>>>
+>>> async def main():
+...     async with Session(auth=AUTH) as sess:
+...         # perform operations here
+...         pass
+...
+>>> asyncio.run(main())
+
+```
+
+Alternatively, use `await Session.aclose()` to close a `Session` explicitly:
+
+```python
+>>> async def main():
+...     sess = Session(auth=AUTH)
+...     # perform operations here
+...     await sess.aclose()
+...
+>>> asyncio.run(main())
+
+```
+
+## Orders Client
+
+The Orders Client mostly mirrors the
+[Orders API](https://developers.planet.com/docs/orders/reference/#tag/Orders),
+with the only difference being the addition of the ability to poll for when an
+order is completed and to download an entire order.
+
+```python
+>>> from planet import OrdersClient
```

Comment:
if you remove the `>>>` and `...` line prefixes, this snippet is easier to copy from the rendered markdown

Reply:
I know, that would be lovely! But as far as I can tell, if I want to doctest the code, each line of code needs to start with `>>>` or `...`. Do you know of a way around this?

Reply:
Lets continue this on #275